### PR TITLE
Implemented Signing and Verifying for Messages

### PR DESCRIPTION
**Pull request by fileninja**
I've implemented the Signing and Verifying Process as follows:
sign_verify

To keep an overview of your signed messages its connected to the local storage of your selected Account. With that you have an list of that where you can delete or copy your signed message.
furthermore you can verify other messages for sure.

#160 is done with that

### CODE REVIEW

File: client/app/src/accounts/AccountController.js [hunk 1]
```diff
@@ -1732,7 +1733,123 @@
       });
 
     }
+    function loadSignedMessages()
+    {
+      self.selected.signedMessages = storageService.get("signedMessages", selectedAccount.signedMessages);
```
Comment (fix) [inline client/app/src/accounts/AccountController.js]: i would more make signed messages fined grained per accounts:
self.selected.signedMessages = storageService.get("signed-"+self.selected.address);

File: client/app/src/accounts/AccountController.js [hunk 2]
```diff
@@ -1732,7 +1733,123 @@
...
+    }
+
+    self.deleteSignedMessage = function(selectedAccount, signedMessage)
+    {
+      var index = selectedAccount.signedMessages.indexOf(signedMessage);
+      selectedAccount.signedMessages.splice(index, index+1);
+      storageService.set("signedMessages", selectedAccount.signedMessages);
```
Comment (fix) [inline client/app/src/accounts/AccountController.js]: storageService.set("signed-"+selectedAccount.address, selectedAccount.signedMessages);

**Comment by fileninja** [comment]
valid ones.. will fix

**Comment by fileninja** [comment]
@fix here you go

**Comment by n4ru** [comment]
God bless!

**Comment by fix** [comment]
+5
closing as i have merged and improved locally

**Comment by sleepdefic1t** [comment]
build/run issues macOS 10.12.x

1.2.2 builds and runs no issues.

1.3.0 after #165; various permissions and dependency issues.

#
node 6.9.2 - Blank Electron Window; permission errors in 'dev tools'  

screen shot  

#
node > 8  
screen shot 2017-07-13 at 10 57 24 pm

**Comment by fileninja** [comment]
@sleepdefic1t this has'nt anything to do with #165 
i also had this before on my ubuntu machine.. seems like an electron error which has something to do with the ledger functionality (node-hid)
somehow i was able to fix it but doesnt know it anymore..
maybe it was this one over here https://stackoverflow.com/questions/39547292/node-module-version-mismatch-expected-50

**Comment by sleepdefic1t** [comment]
Apologies, 
Seemed the likely culprit as last commit.

Tried all that. Searched the issue more. 
That doesn't seem to fix it.
moving this to a separate issue.

Thanks for the help, @fileninja !

**Pull request closed without merging**